ui/app/mainui/activitycenter/panels/ContactRequestCta.qml

```diff
@@ -0,0 +1,56 @@
+import QtQuick 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+import shared.panels 1.0
+
+Item {
+    id: root
+
+    property bool pending: false
+    property bool accepted: false
+    property bool dismissed: false
+    property bool blocked: false
+
+    signal acceptClicked()
+    signal declineClicked()
+    signal blockClicked()
+    signal profileClicked()
+
+    width: buttons.width
+    height: buttons.height
```

Comment:
`implicitFoo` here; and what happens if the `textItem`'s text is bigger than the buttons?

Reply:
fixed